### DISCUSSION TITLE
USB Smart Plug for Construction Timelapse. Schedule when not on Wifi?

### DISCUSSION THREAD

**Discussion by standpointmedia** [gopro/labs#567] [Q&A]
Hi all. 

I saw a recommendation for this Sonoff USB Smart plug on here awhile back. Just wondering if anyones has tested it? I saw one thread mentioning that the schedule timer doesn't work once it's away from the WIFi it was setup on?

https://www.amazon.com/Micro-USB-Smart-Adaptor-Compatible/dp/B083BKBKW5

Thanks!

Declan

[https://imgur.com/a/eKZs0fQ](url)

**Comment by dnewman-gpsw** [comment]
They work, but have a terrible user interface.

**Reply by standpointmedia** [reply]
That's for sure! Thanks. I'll give it a try here so. Any better alternatives out there?
